packages/ui-system/src/components/Button/AnchorButton.tsx

```diff
@@ -8,11 +8,19 @@ interface AnchorButton {
   icon?: React.ReactElement;
 }
 
+const ensureProtocol = (url: string) => {
```

Comment:
**nit:** Let's put it into `utils/routes.ts` as this probably can be reused in the future